### PR TITLE
fix(widget-cfg): hardcode partner fee recipient address

### LINKED ISSUE #3977: [widget] configurator - partner fee recipient nitpick
The configuration currently allows you to define the fee and the recipient of the fee, however, I don't think is super relevant to add the recipient in there.

The configurator is for show case, and I would think is enough to:
- [ ] Show only the BPS of the partnerFee
- [ ] Define ourselves a recipient by default
- [ ] Override in the snippet of code, so we leave a text such us `add-here-the-recipient-of-the-fee`

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes #3977

1. Removed Partner fee recipient input from the widget configurator
2. Hardcoded "0x8BFE784432C91169Bdf3469B52ac589912dE21C3" as a recipient in the widget configurator
3. Added pk of the address into 1password

# To Test

1. The partner fee recipient input should be removed
2. An order app-data with partner fee should contain the address above as a partner fee recipient

### CODE REVIEW

File: apps/widget-configurator/src/app/configurator/consts.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { TradeType, TokenInfo } from '@cowprotocol/widget-lib'
 
 import { TokenListItem } from './types'
 
+export const DEFAULT_PARTNER_FEE_RECIPIENT = '0x8BFE784432C91169Bdf3469B52ac589912dE21C3'
```
Comment (anxolin) [inline apps/widget-configurator/src/app/configurator/consts.ts:6]: Maybe add `0xcA771eda0c70aA7d053aB1B25004559B918FE662` which is CoW Dao

**Review by anxolin — APPROVED** [review]
Great!!

nit, and not even from this PR
image

**Review by elena-zh — APPROVED** [review]
Changes LGTM! 
However, I'd add a form address of the form to fill in into this text 
image

File: apps/widget-configurator/src/app/configurator/consts.ts (hunk 1 above)
Comment (shoom3301) [reply apps/widget-configurator/src/app/configurator/consts.ts:6]: Changed

**Pull request merged**